File: docs/snippets/cpu/dynamic_shape.cpp
#include <openvino/runtime/core.hpp>


int main() {
    {
        //! [static_shape]
        ov::Core core;
        auto model = core.read_model("model.xml");
        ov::Shape static_shape = {10, 20, 30, 40};

        model->reshape(static_shape);
        //! [static_shape]
    }

    return 0;
}

File: docs/snippets/cpu/dynamic_shape.py
# Copyright (C) 2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0


import openvino as ov

#! [static_shape]
core = ov.Core()
model = core.read_model("model.xml")
model.reshape([10, 20, 30, 40])
#! [static_shape]
